app/views/local_transactions/new.html.erb

```diff
@@ -11,22 +11,30 @@
   </p>
 <% end %>
 
-<%= semantic_form_for(@publication, :url => editions_path, :as => :edition, :html => { :id => 'edition-form' } ) do |f| %>
-  <%= f.inputs :lgsl_code %>
-  <%= f.inputs do %>
-    <input type="hidden" name="edition[kind]" value="local_transaction">
-    <%= f.input :panopticon_id, :as => :hidden %>
-    <%= f.input :title, :as => :hidden %>
-    <%= f.input :slug, :as => :hidden %>
-  <% end %>
-  <%= f.inputs :lgil_code %>
-  <%= f.inputs do %>
+<%= form_for(@publication, as: :edition, url: editions_path, html: { id: 'edition-form', novalidate: 'novalidate' } ) do |f| %>
```

Comment:
Just curious what  this `novalidate` does here?

Reply:
It means that the form is not validated when submitted, which is what you need here otherwise the form throws lots of errors that it doesn't have all the information. You fill in that information in the next step